vassal-app/src/main/java/VASSAL/build/module/map/CounterDetailViewer.java

```diff
@@ -589,14 +591,16 @@ else if (displayWhat.equals(EXC_LAYERS)) {
     protected int lastLayer = -1;
     protected int insertPos = 0;
     protected Point foundPieceAt;
+    protected boolean showingOverlap;
 
-    public Visitor(Filter filter, Map map, Point pt) {
+    public Visitor(Filter filter, Map map, Point pt, boolean showOverlap) {
```

Comment:
This could break anything which extends it, or CounterDetailViewer. What you should do here to prevent that is add the following:
```
@Deprecated
public Visitor(Filter filter, Map map, Point pt) {
  this(filter, map, point, false);
}
```
so that the old ctor still exists for anything which calls it.

Reply:
Fixed.